config/config.local.toml

```diff
@@ -1,30 +1,25 @@
 [Log]
 Level = "debug"
-Outputs = ["stdout"]
+Environment = "production"
+Outputs = ["stderr"]
 
 [SyncDB]
 Database = "postgres"
 User = "test_user"
 Password = "test_password"
 Name = "test_db"
-Host = "zkevm-bridge-db"
-Port = "5432"
+Host = "localhost"
+Port = "5435"
 MaxConns = 20
 
-[ClaimTxManager]
-Enabled = true
-FrequencyToMonitorTxs = "1s"
-PrivateKey = {Path = "/pk/keystore.claimtxmanager", Password = "testonly"}
-RetryInterval = "1s"
-RetryNumber = 10
-AuthorizedClaimMessageAddresses = ["0x90F79bf6EB2c4f870365E785982E1f101E93b906"]
-
 [Etherman]
-L1URL = "http://zkevm-mock-l1-network:8545"
-L2URLs = ["http://zkevm-node:8123"]
+L1URL = "https://eth-sepolia.g.alchemy.com/v2/cQPfsPndh1RMnx6MxpdV4XN8Z3EYICsz"
```

Comment:
Should we not store the api key here?

Reply:
We shouldn't 😄 comited by mistake and already rotated.